src/ui/views/ConfiguratorView/index.tsx

```diff
@@ -618,7 +648,14 @@ const ConfiguratorView: FunctionComponent<ConfiguratorViewProps> = (props) => {
   }, [buildInProgress, response]);
 
   const onBuild = () => sendJob(BuildJobType.Build);
-  const onBuildAndFlash = () => sendJob(BuildJobType.BuildAndFlash);
+  const onBuildAndFlash = () => {
+    if (forceFlash) {
+      sendJob(BuildJobType.ForceFlash);
```

Comment:
I think this is a good time to get rid of "ForceFlash" build job type. `force` and `erase` should be boolean flags. `BuildJobType` should retain `Build` and `Flash` only.